README.yaml

```diff
@@ -0,0 +1,125 @@
+---
+#
+# This is the canonical configuration for the `README.md`
+# Run `make readme` to rebuild the `README.md`
+#
+# Name of this project
+name: terraform-aws-cloudwatch-agent
+
+# Tags of this project
+tags:
+  - aws
+  - terraform
+  - terraform-modules
+  - platform
+  - cloudwatch
+  - agent
+  - cloudwatch-agent
+
+# Categories of this project
+categories:
+  - terraform-modules/platform
+
+# Logo for this project
+#logo: docs/logo.png
+
+# License of this project
+license: "APACHE2"
+
+# Canonical GitHub repo
+github_repo: cloudposse/terraform-aws-cloudwatch-agent
+
+# Badges to display
+badges:
+  - name: "Build Status"
+    image: "https://travis-ci.org/cloudposse/terraform-aws-cloudwatch-agent.svg?branch=master"
+    url: "https://travis-ci.org/cloudposse/terraform-aws-cloudwatch-agent"
+  - name: "Latest Release"
+    image: "https://img.shields.io/github/release/cloudposse/terraform-aws-cloudwatch-agent.svg"
+    url: "https://github.com/cloudposse/terraform-aws-cloudwatch-agent/releases/latest"
+  - name: "Slack Community"
+    image: "https://slack.cloudposse.com/badge.svg"
+    url: "https://slack.cloudposse.com"
+
+related:
```

Comment:
can also add these CloudWatch-related modules to `related` (good for search and SEO):

https://github.com/cloudposse?utf8=%E2%9C%93&q=cloudwatch-&type=&language=

Reply:
:+1: